src/pools/D3MAavePool.sol

```diff
@@ -74,11 +115,17 @@ interface RewardsClaimerLike {
 
 contract D3MAavePool is ID3MPool {
 
+    enum AaveVersion {
+        V2,
+        V3
+    }
+
     mapping (address => uint256) public wards;
     address                      public hub;
     address                      public king; // Who gets the rewards
     uint256                      public exited;
 
+    AaveVersion     public immutable version;
     bytes32         public immutable ilk;
     VatLike         public immutable vat;
     LendingPoolLike public immutable pool;
```

Comment:
Same here, suggest to make `pool` an address and cast upon usage.

Reply:
[done](https://github.com/makerdao/dss-direct-deposit/pull/93/commits/17c7a15b8706ed516f58f7de63e64e5a1d3701f8)